content/post-quantum-2025.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Sigstore & Post-Quantum Cryptography (2025)"
+author = "Zach Steindler (GitHub)"
+date = "2025-05-15"
+tags = ["sigstore", "nist", "postquantumcryptography"]
+type = "post"
+draft = false
++++
+
+In the coming years, systems will transition to post-quantum cryptographic algorithms (PQCA). There is some inherent tension in these transitions as we learn things through adoption, but making decisions too soon can saddle you with tech debt. The quick summary is that the Sigstore project wants to enable people to sign content with PQCA keys as soon as possible, and adopt PQCA in the Sigstore services (like Fulcio, Rekor, and a timestamp authority) when reliable and vetted PQCA is available in the Go ecosystem.
+
+![klystron gallery at Standford linear accelerator](/images/slac.jpg)
+
+This document will describe what's happening with PQCA outside of Sigstore, give an expected timeline for PQCA adoption in Sigstore, and talk about open questions raised by the transition.
+
+There's a [discussion in the sigstore/community repository](https://github.com/sigstore/community/discussions/591) where you can respond to the content in this post.
+
+### Definitions
+
+Because this work is in progress, there are synonyms for the same concept coming from the academic community or writing a US government standard.
```

Comment:
Looks good, but "or writing a US government standard." might need a tweak, hard to grok the meaning here.